OMPython/docker/Dockerfile.ompython.linux64.focal

```diff
@@ -0,0 +1,35 @@
+# Build with
+# $ docker build --tag docker.openmodelica.org/ompython:focal -f Dockerfile.ompython.linux64.focal .
+FROM ubuntu:focal
+
+# Get rid of the warning: "debconf: unable to initialize frontend: Dialog"
+# https://github.com/moby/moby/issues/27988
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# Install git, wget, Python 3 + pandas
+RUN apt-get update \
+    && apt-get install git python3-pip wget -qy \
+    && pip3 install pandas \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get clean
+
+# Install OMC and OMPython
+RUN for deb in deb deb-src; do echo "$deb http://build.openmodelica.org/apt focal stable"; done | tee /etc/apt/sources.list.d/openmodelica.list \
```

Comment:
This should use a fixed version, like https://build.openmodelica.org/omc/builds/linux/releases/1.16.2/

But you could just use `FROM openmodelica/openmodelica:v1.16.2-ompython` instead, right? And only install pandas to it.

Reply:
We could do that. Where is the dockerfile for that one saved? I think we should have them all in one place.

Reply:
https://github.com/OpenModelica/OpenModelicaDockerImages uses Docker Hub's settings to build from this repository's branches. It's pretty slow and error-prone though, and I want to eventually move it to Jenkins.